src/Constants.php

```diff
@@ -324,10 +324,10 @@ class Constants
         'craft\fields\Money',
     ];
 
-    const CTA_PROFESSIONAL_TRANSLATION = "https://www.acclaro.com/get-started/translation-quote/";
-    const CTA_QUALITY_REVIEW = "https://www.acclaro.com/localization/";
-    const CTA_TECHNICAL_SUPPORT = "https://www.acclaro.com/localization-partner/";
-    const CTA_AI_ML_TRANSLATION = "https://www.acclaro.com/localization/";
+    const CTA_PROFESSIONAL_TRANSLATION = "https://www.acclaro.com/solutions/global-marketing/website-localization-services";
+    const CTA_QUALITY_REVIEW = "https://www.acclaro.com/solutions/language-quality";
+    const CTA_TECHNICAL_SUPPORT = "https://info.acclaro.com/craftcms-technical-support";
+    const CTA_AI_ML_TRANSLATION = "https://info.acclaro.com/translation-plugin-for-craft3-support-info";
```

Comment:
@bhupeshappfoster @sharmarishabh1 are these constants also used for the services page?

Also, the value for `CTA_AI_ML_TRANSLATION` is incorrect. We can just have an empty string for now.

Reply:
@sidedwards  Yes, we are using the same on the services page too. 

@bhupeshappfoster 
Can you please add "#" here instead?

Reply:
Done